lib/Mandel/Document.pm

```diff
@@ -240,6 +240,15 @@ sub import {
 
 sub TO_JSON { shift->data }
 
+sub _validate_fields {
```

Comment:
I'm not sure if this should be a private method. What do you think? Could it be useful to call that after loading data from the store?

Reply:
My original thinking was, that if the module does its job correctly, there is no need for a user to call the method. I didn't think of the case, that something else might modify the data while it's stored in database. I made the method now public and documented it.